css/main.css

```diff
@@ -0,0 +1,398 @@
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1vh;
+  padding: 15vh 0 25vh 0;
+  border-bottom: 1px solid black;
+}
+
+.contact-header {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #182d50;
+}
+
+form {
+  display: flex;
+  gap: 2vh;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#name,
+#email,
+#text {
+  width: 100%;
+}
+
+.label1,
+.label2,
+.label3 {
+  padding: 0 10vw;
+}
+
+#name,
+#email {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-color: #d3d0cf;
+  height: 7vh;
+}
+
+#text {
+  background-color: cornsilk;
+  border: none;
+  padding-top: 2vh;
+  padding-left: 1vw;
+}
+
+.label4 {
+  display: flex;
+  justify-content: center;
+  padding: 2vh 0 5vh 0;
+}
+
+#submit {
```

Comment:
Let us make the cursor of the submit button a [pointer.](https://www.w3schools.com/cssref/pr_class_cursor.asp), The cursor property in CSS controls what the mouse cursor will look like when it is located over the element in which this property is set. This improves the UX of your application.

Reply:
Alright. I will do just that. Thanks you :pray: